config/locales/client.en.yml

```
@@ -3016,6 +3016,10 @@ en:
       recent: "Recent Searches"
       clear_recent: "Clear Recent Searches"
 
+      experience:
+        search_field: "Search field in site header"
+        search_icon: "Search icon by profile avatar"
```

```suggestion
        search_icon: "Search icon near profile avatar"
```

Maybe this? Minor but I think it makes slightly more sense